src/presets/netlify/utils.ts

```diff
@@ -116,6 +116,8 @@ export const config = {
   name: "server handler",
```

Comment:
@serhalp (unrelated) from [docs](https://docs.netlify.com/frameworks-api/#configuration-options-2) i noticed it should be `displayName`? 🤔 (are there some types we could reuse perhaps for this object?)